cardano-api/golden/Test/Golden/ErrorsSpec.hs

```diff
@@ -295,3 +302,15 @@ spec = describe "Test.Golden.Errors" $ do
   it "TxMetadataRangeError" $ H.requireTest $ do
     flip H.diffVsGoldenFile "golden/files/errors/TxMetadataRangeError/TxMetadataJsonToplevelNotMap.txt"
       $ displayError $ TxMetadataBytesTooLong 0
+
+testErrorMessage :: (HasCallStack, MonadIO m, MonadTest m, Data p , Error p) => p -> m ()
```

Comment:
I don't see much benefit of introducing orphan instances vs hard coding the golden file name.

Reply:
:+1:  cc: @newhoggy

Reply:
I think I'm okay with this as long as we attempt to reduce the amount of orphaned instances by moving them to where the types are defined wherever possible and other move them to `Orphan` modules.